<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>administrate_position</class>
 <widget class="QWidget" name="administrate_position">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Адміністрування посад</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>331</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Назва посади</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="position_picker"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Зарплата</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="salary">
        <property name="inputMask">
         <string notr="true">90000000000000000000</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Кількість місць</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="position_quantity">
        <property name="inputMask">
         <string>90000000000000000000</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Дата початку</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="start_date">
        <property name="minimumDate">
         <date>
          <year>1770</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
        <property name="displayFormat">
         <string>yyyy-M-d</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="submit_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>331</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Внести дані</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
